data/posts/practicalities.markdown

```diff
@@ -1,28 +1,33 @@
  - autem: Parkujte prosím [zde](https://www.google.com/maps/@49.9265943,14.2752967,3a,75y,237.21h,82.98t/data=!3m7!1e1!3m5!1soLjz43xmoga19bxuKs3q3A!2e0!6s%2F%2Fgeo1.ggpht.com%2Fcbk%3Fpanoid%3DoLjz43xmoga19bxuKs3q3A%26output%3Dthumbnail%26cb_client%3Dmaps_sv.tactile.gps%26thumb%3D2%26w%3D203%26h%3D100%26yaw%3D236.34343%26pitch%3D0%26thumbfov%3D100!7i13312!8i6656). 
 
-Do Hostími se dá dostat:
- - vlakem do Srbska: 40 minut z Prahy, z nádraží je to krásná procházka Českým Krasem (3.5 km), vyznačeno [na mapě](https://drive.google.com/open?id=1JaI7MwUYkx4QFXDBjTwGt7zRSitqprZQ&usp=sharing). Budeme mít také zajištěno jedno auto, které bude pendlovat mezi Hostímí a Srbskem (do noci).
- - autobusem: 50 min z Prahy, ale jenom párkrát za den
- - autem: trasa ze Dobřichovic vyznačená [na mapě](https://drive.google.com/open?id=1JaI7MwUYkx4QFXDBjTwGt7zRSitqprZQ&usp=sharing]), parkujte před hospodou U Krobiána
+Do Hostími se dostanete:
+ - vlakem do Srbska: 40 minut z Prahy, z nádraží je to krásná procházka Českým krasem (3,5 km), vyznačeno [na mapě](https://drive.google.com/open?id=1JaI7MwUYkx4QFXDBjTwGt7zRSitqprZQ&usp=sharing). Budeme mít také zajištěno jedno auto, které bude pendlovat mezi Hostímí a Srbskem (do noci),
+ - autobusem: 40 min z Prahy, ale jenom [párkrát za den](https://mapy.cz/zakladni?x=14.2905668&y=50.0545102&z=17&timeboard=%7B%22l%22%3A%22100384%22%2C%22d%22%3A-1%2C%22i%22%3A1%2C%22v%22%3A%7B%22from%22%3A%222018-05-18T00%3A00%3A00%22%2C%22to%22%3A%222018-08-16T00%3A00%3A00%22%7D%7D&source=pubt&id=15305993),
+ - autem: trasa z Dobřichovic vyznačená [na mapě](https://drive.google.com/open?id=1JaI7MwUYkx4QFXDBjTwGt7zRSitqprZQ&usp=sharing]), parkujte před hospodou U Krobiána.
 
 # Ubytování
 
-V Hostími můžete určitě přespat, možností je víc. Seznam možností je v [registraci](https://docs.google.com/forms/d/e/1FAIpQLSdbxmrXbRi36ecU4XsamYie2bFquFw-GankxrnCPdJTNU0lTA/viewform?usp=sf_link). Vyplňte prosím registraci, ať můžeme pokoje zarezervovat.
+V Hostími můžete přespat, možností je víc. Seznam možností je v [registraci](https://docs.google.com/forms/d/e/1FAIpQLSdbxmrXbRi36ecU4XsamYie2bFquFw-GankxrnCPdJTNU0lTA/viewform?usp=sf_link). Vyplňte prosím registraci, ať můžeme pokoje zarezervovat.
 
 # Co na sebe?
 
 Na obřad si vemte něco formálnějšího, ale nemusíte se pařit v saku, košile bohatě stačí. Na oslavu si vemte cokoliv, co vám je pohodlné. 
 
-Naše oblíbená barva je fialová, tak budete s námi lépe ladit nějakým fialovým doplňkem. Není to ale vůbec potřeba :)
+Naše oblíbená barva je fialová, tak s námi budete lépe ladit nějakým fialovým doplňkem. Není to ale vůbec potřeba :)
+
+# Papání
+
+Před obřadem od nás dostanete nějakou tu jednohubku, ale radši si dejte doma i snídani. Oběd bude rozdělen na oběd pro nejbližších 20 příbuzných a oběd pro ostatní desítky hostů. Snad nám nebudete vyčítat, že neuvidíte, jak se pobryndáme polévkou :)
```

Comment:
"pro ostatní desítky" je divné, napsal bych rovnou "pro ostatní". Nenapíšeme rovnou, že bude "boršč na sále", ať nečekají nic extra? Nebo to upřesníme později?